runtime/onert/api/include/nnfw.h

```diff
@@ -177,6 +177,8 @@ NNFW_STATUS nnfw_close_session(nnfw_session *session);
 /**
  * @brief     Load model from nnpackage file or directory
  *
+ * The length of \p package_file_path must not execeed 1024 bytes including zero at the end.
```

Comment:
Not sure which value is good for the path length limit.

`PATH_MAX` values in OSes:

Windows : 260
macOS : 1024
Linux : 4096

Reply:
We have developed for Linux base system but haven't tried Windows... 1024 seems fine for me. :-)